nixos/modules/services/monitoring/netdata.nix

```diff
@@ -78,6 +83,21 @@ in {
         };
       };
 
+      extraPluginPaths = mkOption {
+        type = types.listOf types.path;
+        default = [ ];
+        example = literalExample ''
+          [ "/path/to/plugins.d" ]
+        '';
+        description = ''
+          Extra paths to add to the netdata global "plugins directory"
+          option.  Useful for when you want to include your own
+          collection scripts.
```

Comment:
Can you add a link to some docs regarding that?

Reply:
Unfortunately this is terribly documented. Let me look at the upstream docs and see if I can find anything useful.

Reply:
@Infinisil Unfortunately it's not a straight forward process.  You can see from this [package I wrote](https://github.com/pjones/phoebe/blob/eab7c456171f0ec0713cea5460cbddc4d74709c5/modules/services/monitoring/plugins.nix) what I had to do to get netdata to run my own plugin.  

Is there a better place than in this option to document the process?  Perhaps if I write a NixOS test that includes a package similar to the one I linked to we can add a reference to it in this option documentation.  What do you think about that?

Reply:
> Let me look at the upstream docs and see if I can find anything useful.

Found anything?

If it's badly documented that's not our problem, we can't be expected to write docs for upstream. I think best thing to do is to file an issue upstream regarding this.

Reply:
@Infinisil I added a link to some documentation for writing a custom plugin.  This should be ready to merge now.